### ISSUE TITLE
dont display hashtag and post sections when searching for an account

### ISSUE THREAD

**Issue by wakest** [cheeaun/phanpy#513]
when searching for a url, the second you have a result you know that it isn't a hashtag or a post, so it makes sense to then not show those categories after phanpy knows what its searching for...

<img width="645" alt="image" src="https://github.com/cheeaun/phanpy/assets/7890201/b26c071a-0521-428e-97c1-429213cf2242">

**Version**
https://dev.phanpy.social/ build 2024.04.19.7376cb1
Mastodon account: https://social.wake.st/@liaizon
macOS 13.6.4
Firefox 124.0.2 (64-bit)

**Comment by cheeaun** [comment]
> it makes sense to then not show those categories

This is subjective. Hiding them also means there's no (visual) hint that no results are found for those categories and can be thought as a bug.

**Comment by wakest** [comment]
if you search for a link there can never be results for a hashtag right? so why would you need a hint that theres no results for something that there can never be results for?

**Comment by cheeaun** [comment]
> if you search for a link there can never be results for a hashtag right?

True, but still depends on the backend 😉 (they could return _something_? 😂). I know we can try to be smart about this but as of now, I'll keep this in mind and continue observing user behaviour/feedback. This search page is still "under-designed" and there are various ways to further improve it.
